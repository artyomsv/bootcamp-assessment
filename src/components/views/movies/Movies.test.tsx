import * as React from 'react';
import {createShallow} from '@material-ui/core/test-utils';
import MoviesView from './Movies.view';

const shallow = createShallow();

describe('renders without crashing', () => {
  test('if renders', () => {
    const mockFn = jest.fn();
    const result = shallow(
      <MoviesView
        movies={[]}
        isFetching={false}
        fetchMovies={mockFn}
        navigateToMovie={mockFn}
        page={{page: 1, totalPages: 10, totalResults: 200}}
        search={''}
        onQueryChange={mockFn}
      />
    );

    expect(result).toMatchSnapshot();
  });
});
