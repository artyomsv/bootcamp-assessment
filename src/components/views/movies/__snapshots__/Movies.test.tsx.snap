// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing if renders 1`] = `
<ActorsView
  classes={
    Object {
      "textField": "ActorsView-textField-2",
      "top": "ActorsView-top-1",
    }
  }
  fetchMovies={[MockFunction]}
  isFetching={false}
  movies={Array []}
  navigateToMovie={[MockFunction]}
  onQueryChange={[MockFunction]}
  page={
    Object {
      "page": 1,
      "totalPages": 10,
      "totalResults": 200,
    }
  }
  search=""
/>
`;
